crates/rethnet_eth/src/signature.rs

```diff
@@ -346,4 +379,50 @@ mod tests {
 
         assert_eq!(s1, s2);
     }
+
+    #[test]
+    fn test_private_key_to_address() {
+        // `hardhat node`s default addresses are shown on startup. this is the first one:
+        //     Account #0: 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266 (10000 ETH)
+        //     Private Key: 0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
+        // we'll use these as fixtures.
+
+        use std::str::FromStr;
```

Comment:
Put it at the top of `mod tests`. There is not performance decrement and it is in line with the auto-include functionality of Rust Analyzer's formatting

Reply:
addressed in https://github.com/NomicFoundation/hardhat/pull/4149/commits/8e67e4339e20771175c71de84669ccc1d7047ee7